packages/message-manager/src/PersonalMessageManager.test.ts

```diff
@@ -1,6 +1,31 @@
+import { detectSIWE } from '@metamask/controller-utils';
 import { PersonalMessageManager } from './PersonalMessageManager';
 
+jest.mock('@metamask/controller-utils', () => ({
+  ...jest.requireActual('@metamask/controller-utils'),
+  detectSIWE: jest.fn(),
+}));
+
+const siweMockNotFound = {
+  isSIWEMessage: false,
+  parsedMessage: null,
+};
+
+const siweMockFound = {
+  isSIWEMessage: true,
+  parsedMessage: {
+    address: '0x0000000',
+    domain: 'example.eth',
+  },
+} as any;
```

Comment:
Nit: This cast doesn't seem to be required
```suggestion
};
```